### ISSUE TITLE
Display data panel in a floating window ?

### ISSUE THREAD

**Issue by Mavialle** [3liz/lizmap-web-client#2159]
Hello,

Is there a way to display data panel in a floating window ?

Many thanks for your help.

### Environment
 - Lizmap version: 3.4.1
 - QGIS Server version: ≥ 3.0
 - QGIS Desktop FULL version: 3.10.10
 - OS (Windows, Linux, MacOS, Android…): Windows
 - Browser (Firefox, Chrome…): Firefox or Chrome
 - Lizmap admin information (users, logs…) stored into Sqlite (default)? PostgreSQL? MySQL? I don't know

**Comment by nboisteault** [comment]
Hi,
This is not possible but you could be developed.

**Comment by Mavialle** [comment]
Hi,

in what way? javascript ?

**Comment by nboisteault** [comment]
Yes it would need some JavaScript. Jquery-UI dialog could do the job i guess.
